src/providers/README.md

```diff
@@ -2,3 +2,85 @@
 
 This directory contains providers contributing additional bindings, for example
 custom sequence actions.
+
+## Overview
+
+A provider is a class that provides a `value()` function. This function is called `Context` when another entity requests a value to be injected.
+
+Here we create a provider for a logging function that can be used in a custom sequence action.
+
+The logger will log the URL, the parsed request parameters, and the result. The logger is also capable of timing the sequence if you start a timer at the start of the sequence using `process.hrtime()`.
+
+## Basic Usage
+
+### LogProvider
+
+LogProvider can automatically be bound to your Application's Context using the LogComponent which exports the provider with a binding key of `extension-starter.actions.log`. You can learn more about components in the [related resources section](#related-resources). 
+
+The key can be accessed by importing `ExtensionStarterBindings` as follows:
+**Example: Binding Keys**
+```
+import {ExtensionStarterBindings} from 'HelloExtensions';
+// Key can be accessed as follows now
+const key = ExtensionStarterBindings.LOG_ACTION;
+```
+
+LogProvider is best used as a Sequence Action. You can define your own sequence action as shown below. 
```

Comment:
> You can define your own sequence action as shown below. 

This does not match my understanding of terms "sequence" and "sequence action". A custom sequence is a class that defines the order in which various sequence actions (request-handling steps) are invoked.

In order to define an own sequence action, one has to write the function implementing it (in essence), or a provider that can create such function (in practice).

To leverage such own sequence action, one has to write their own sequence (class), this class can use both built-in actions like "findRoute" and custom actions contributed by extensions.

So, I am proposing to reword this to "you can use your own sequence action as shown below". Or perhaps "in order to use your own sequence action, you should define your own sequence as shown below".

Reply:
Thank you for clarifying. I had been using Sequence Action & Sequence interchangeably which would've certainly been confusing for new users.